### PR TITLE
Assimilate 2.0.0 to Rally

### PR DESCRIPTION
**Pull request by jeffmay**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,6 +19,89 @@ implicits and tools for:
 - Compile-time Json.oformat and Json.owrites macros (Play 2.3 only)
 
 <h1>Versions</h1>
+
```
Comment (aman-rally) [inline README.md]: Why is the read me HTML instead of markup?

File: play23-json-ops/src/main/scala/play/api/libs/json/ops/Exceptions.scala [hunk 2]
```diff
@@ -12,7 +12,7 @@ private[ops] object Exceptions {
     */
   def keyAsCodeLiteral(keyValue: Any): String = {
     keyValue match {
-      case s: String => "\"%s\"".format(s) // for some reason string interpolation goes wonky here in 2.10
+      case s: String => "\"%s\"".format(s) // for some reason string interpolation goes wonky here in Scala 2.11
```
Comment (aman-rally) [inline play23-json-ops/src/main/scala/play/api/libs/json/ops/Exceptions.scala]: This is also true with 2.12. I believe this is a property of string interpolation.
`s""" "$s" """.trim` is another alternative if you want to use it

File: play23-json-ops/src/main/scala/play/api/libs/json/ops/JsonTransform.scala [hunk 3]
```diff
@@ -29,7 +29,7 @@ object JsonTransform {
    *
    * @return a transformer that will pass the json through unchanged
    */
-  implicit def defaultTransform[A: ClassTag]: JsonTransform[A] = JsonTransform[A]((x: JsValue) => x)
+  implicit def defaultTransform[A: ClassTag]: JsonTransform[A] = JsonTransform[A](identity[JsValue])
```
Comment (macmacbr) [inline play23-json-ops/src/main/scala/play/api/libs/json/ops/JsonTransform.scala]: 👍

File: README.md (hunk 1 above)
Comment (jeffmay) [reply README.md]: When I first started using this pattern, I didn't know how to do it in Markdown. It looks like GitHub has a newer markdown version which supports images in tables, so I'll have to give that a shot.

**Pull request merged**